linchpin/cli/__init__.py

```diff
@@ -471,7 +472,7 @@ def lp_validate(self, targets=(), old_schema=False):
         return self.do_validation(prov_data, old_schema=old_schema)
 
 
-    def lp_destroy(self, targets=(), run_id=None, tx_id=None):
+    def lp_destroy(self, targets=(), run_id=None, tx_id=None, vault_pass=''):
```

Comment:
See the reference above for `do_action`. `vault_pass` should not need to be passed, use the context object instead.